local/blockchain.go

```diff
@@ -104,19 +105,20 @@ func (ln *localNetwork) RegisterBlockchainAliases(
 	chainInfos []blockchainInfo,
 	chainSpecs []network.BlockchainSpec,
 ) error {
-	fmt.Println()
+	fmt.Println() // TODO can we remove all these empty line prints?
```

Comment:
this is old heritage. improves a little the visual. I am not opposite. What do you think @dasconnor @holisticode ?

Reply:
A run produces a lot of logs. It is often pretty difficult to find things. I like having some visual separation of things. 

But then, I usually need to look for logs in the log file, and these lines aren't put into the log files? Therefore I am indifferent to this issue. Whatever you prefer.

Reply:
Personally I'd prefer to remove but up to you guys. Let me know either way and I'll either remove the TODO or the `fmt.Println()`

Reply:
I'm on the fence as well, I'd say let's just leave them.

Reply:
Pls remove the TODO

Reply:
done